communication/mavsdkvehicleserver.cpp

```diff
@@ -1,14 +1,17 @@
 /*
- *     Copyright 2022 RISE Research Institutes of Sweden AB, Safety and Transport   waywise@ri.se
+ *     Copyright 2025 RISE Research Institutes of Sweden AB, Safety and Transport   waywise@ri.se
  *     Published under GPLv3: https://www.gnu.org/licenses/gpl-3.0.html
  */
 
-#include "mavsdkvehicleserver.h"
 #include <QDebug>
 #include <QMetaMethod>
 #include <future>
 #include <algorithm>
 #include <chrono>
+#include <gpiod.hpp>
```

Comment:
MavsdkVehicleServer should not handle vehicle hardware specifics (this include should not be here) IMO. Instead, it should delegate to classes implementing the specifics.